radiomics/imageoperations.py

```diff
@@ -248,7 +239,117 @@ def applyThreshold(inputImage, lowerThreshold, upperThreshold, insideValue=None,
   return tif.Execute(inputImage)
 
 
-def swt3(inputImage, wavelet="coif1", level=1, start_level=0):
+def applyFilterOriginal(inputImage, **kwargs):
```

Comment:
As a historical perspective, @hugoaerts specifically didn't like the word "filter" as applied to this kind of operations, and that is why the name of the module is `imageoperations`. I personally agree that "filter" is a more recognizable and conventionally used term.